ra_aid/callbacks/default_callback_handler.py

```
@@ -329,14 +329,16 @@ def _handle_callback_update(
             # Must Convert Decimal to float compatible JSON serialization in repository
             cost_float = float(cost)
 
-            self.trajectory_repo.create(
+            trajectory_record = self.trajectory_repo.create(
```

Ruff (0.8.2)

Local variable `trajectory_record` is assigned to but never used

Remove assignment to unused variable `trajectory_record`

(F841)